Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/ICounterValue.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation
+{
+    using Microsoft.ApplicationInsights.DataContracts;
+
+    /// <summary>
+    /// Interface represents the counter value.
+    /// </summary>
+    public interface ICounterValue
```

Comment:
I really liked when we were returning metric telemetry as I can return value and it's base, not just a value. Not sure whether we did it in any of the gauges.